### ISSUE TITLE
1.1.29: Unit test 'core.files.objfile' assertion has failed at file utest/core/files/objfile.cpp, line 37

### ISSUE THREAD

**Issue by dvzrv** [sadko4u/lsp-plugins#179]
Hi! When building 1.1.29 for Arch Linux, I ran the test suite against it again.

This fails unfortunately:

```
Unit test 'core.files.objfile' assertion has failed at file utest/core/files/objfile.cpp, line 37:
  res == STATUS_OK
  Error loading scene from file res/test/3d/parse/parse.obj
```

I run the build and tests as follows:

```
make
make test
./.test/lsp-plugins-test utest --nofork --debug --verbose
```

The complete build log can be found here: 
[lsp-plugins-1.1.29-build.log](https://github.com/sadko4u/lsp-plugins/files/5835571/lsp-plugins-1.1.29-build.log)

**Comment by sadko4u** [comment]
Yes, can confirm the problem. Shame on me.

**Comment by sadko4u** [comment]
Pushed changes to **devel** branch: e4d9578d222d42c54e9928db4866e749bafe1edd
and to **hotfix** branch: 93caf7b75d0ca727844f9228380684aab0289402

**Comment by dvzrv** [comment]
Haha, no worries. Bugs happen :)

**Comment by dvzrv** [comment]
Also, thanks for fixing it so fast. I can patch the release with this, no problem!

**Comment by sadko4u** [comment]
@dvzrv No, shame on me that I didn't run unit tests before release.

**Comment by sadko4u** [comment]
Available in 1.1.30
